docs/enrollment_and_payment.md

```diff
@@ -0,0 +1,138 @@
+## Enrolling a card
+
+In order to perform a payment a card first needs to be enrolled and tokenized. First you must gather the necessary information to perform an enrollment. This data is then sent to the ePayment Platform for tokenization.
```

Comment:
> In order to perform a payment a card first needs to be enrolled and tokenized.

Not sure if I should comment on stylistics, but I struggled to read this sentence. I suggest:

In order to perform a payment, a card needs to be enrolled and tokenised first.

Reply:
Agreed, adjusted!